diff_cover/violations_reporter.py

```diff
@@ -116,14 +116,14 @@ def _get_classes(self, xml_document, src_path):
              src_abs_path in [
                  os.path.join(
                      source,
-                     clazz.get('filename')
+                     os.path.normpath(clazz.get('filename').replace("\\", "/"))
                  ) for source in sources]]
             or
             [clazz for clazz in classes if
-             clazz.get('filename') == src_abs_path]
+             os.path.normpath(clazz.get('filename').replace("\\", "/")) == src_abs_path]
             or
             [clazz for clazz in classes if
-             clazz.get('filename') == src_rel_path]
+             os.path.normpath(clazz.get('filename').replace("\\", "/")) == src_rel_path]
```

Comment:
Oh, I think `os.path.normpath` is needed for the example where the path begins with `.\`. Perhaps you could use `posixpath.normpath(clazz.get('filename').replace("\\", "/"))` if you wanted to keep doing something similar that will run on your machine too.